src/error_gui.cpp

```diff
@@ -300,12 +303,15 @@ struct ErrmsgWindow : public Window, ErrorMessageData {
 		if (_right_button_down && this->duration != 0) delete this;
 	}
 
-	void OnHundredthTick() override
+	void OnRealtimeTick(uint delta_ms) override
 	{
+		if (!this->duration_timer.Elapsed(delta_ms)) return;
```

Comment:
While here, maybe better to set the `duration_timer` to the correct interval from the start, and run it only once? Instead of the additional `duration` variable?